what-the-stack/src/main/java/com/haroldadmin/whatthestack/WhatTheStackActivity.kt

```diff
@@ -31,11 +31,11 @@ class WhatTheStackActivity : AppCompatActivity() {
         setContentView(R.layout.activity_what_the_stack)
 
         window.decorView.systemUiVisibility =
-            View.SYSTEM_UI_FLAG_LAYOUT_HIDE_NAVIGATION or View.SYSTEM_UI_FLAG_LAYOUT_STABLE
+                View.SYSTEM_UI_FLAG_LAYOUT_HIDE_NAVIGATION or View.SYSTEM_UI_FLAG_LAYOUT_STABLE
 
         Insetter.builder()
-            .applySystemWindowInsetsToPadding(Side.LEFT or Side.RIGHT or Side.TOP)
-            .applyToView(findViewById(R.id.nestedScrollRoot))
+                .applySystemWindowInsetsToPadding(Side.LEFT or Side.RIGHT or Side.TOP)
+                .applyToView(findViewById(R.id.nestedScrollRoot))
```

Comment:
Please refrain from modifying unrelated lines. Sometimes it's necessary, but not in this case.

Reply:
done